app/renderer/components/ServerMonitor/ServerMonitor.js

```diff
@@ -115,12 +115,13 @@ export default class ServerMonitor extends Component {
     document.removeEventListener('keydown', this.keydownListener);
   }
 
-  componentWillUpdate () {
+  getSnapshotBeforeUpdate () {
```

Comment:
I don't see the connection between the name of this method and what it is actually doing
Perhaps, we could set a better name to it?